students/robert/Session 4/mailroom.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python 
+
+def main_command():
+    while True:
+        user_choice = input("Type 1 for Send a Thank You; Type 2 for Create Report: Type 3 to quit: ->")
+        try:
+            user_choice = int(user_choice)
+            if user_choice not in [1, 2, 3]:
+              print("Invalid selection. Please try again.")
+            else:
+              break
+        except:
+            print("Invalid selection. Please try again.")
+        finally:
+            pass
+    return user_choice
+
+
+
+def donor_name():
+    for i in dln.keys():
+        print(i) 
+
+def Sending_a_Thank_You():
```

Comment:
It's good practice, and very helpful for both you and other coders, if you add comments to your code describing what some of the different pieces do...